builder/proxmox/common/config.go

```diff
@@ -280,6 +281,9 @@ func (c *Config) Prepare(upper interface{}, raws ...interface{}) ([]string, []st
 		if c.NICs[idx].Model != "virtio" && c.NICs[idx].PacketQueues > 0 {
 			errs = packersdk.MultiErrorAppend(errs, fmt.Errorf("network_adapters[%d].packet_queues can only be set for 'virtio' driver", idx))
 		}
+		if (c.NICs[idx].MTU < 0) || (c.NICs[idx].MTU > 65520) {
```

Comment:
Given we are performing multiple checks here on each NIC, we could probably change the way we iterate on those by using a `for idx, nic := range c.NICs`, this way we can access the values without having to work with the slice directly.

Reply:
This can be done in a separate commit/PR though, no need to amend the existing ones

Reply:
That's a great suggestion - I guess that would also make sense for at least some of the iterations on `c.Disks` and `c.AdditionalISOFiles` as well. If so, I can change these as well and open a separate PR on that.

Reply:
Oh true, if we're to fix all of them, might as well do them in a later PR then; I'll merge this one in the meantime, feel free to come back to the loops later, thanks @sebastian-de.